test/web-worker.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line no-undef
+importScripts('/base/amplitude.js');
+
+var isTrue = function (a) {
+  if (!a) {
+    throw new Error('Assertion failed: object is falsey.');
+  }
+};
+
+describe('web worker', function () {
+  describe('init', () => {
+    it('should call init successfully', () => {
+      let successCallbackCalled = false;
+      let errorCallbackCalled = false;
+      amplitude.init(
+        'API_KEY',
+        undefined,
+        {
+          onError: function onError() {
```

Comment:
Or import sinon?

Reply:
`sinon.js` can work with `importScripts` and was able to add that.